apps/src/p5lab/P5LabView.jsx

```diff
@@ -4,8 +4,18 @@ import classNames from 'classnames';
 import {connect} from 'react-redux';
 import PropTypes from 'prop-types';
 import React from 'react';
-import AnimationTab from './AnimationTab/AnimationTab';
+
+import experiments from '@cdo/apps/util/experiments';
```

Comment:
This is all just moving around imports to match our current preferred import order/categories, with the exception of adding the `experiments` import